### ISSUE TITLE
Address Additional Data Verification Non-critical Accessibility Issues

### ISSUE THREAD

**Issue by governmentSponsored** [US-Trustee-Program/Bankruptcy-Oversight-Support-Systems#1093]
This ticket covers accessibility issues beyond those required by WCAG.

These issues were originally on CAMS-438, but were not addressed at the time 438 was done because further information was needed that was not available at the time.

See google document for CAMS-438-accessibility notes which contain comments and questions regarding these issues.

- [x] Numerous elements overlap awkwardly when viewport is zoomed by 200%. Tables get so wide that some fields go offscreen, cannot see anymore. Will need a one-column implementation (requires design update).

Investigate the issues identified with screen reader below (VoiceOver on Flexion, may need to verify with Narrator).

Screen Reader Issues
- [ ] Each case event row is read as “Heading level 4, court district southern district of new york order filed zero six slash twenty slash two-thousand ten event type consolidation event status pending review, collapsed, button, group.” That is extremely verbose. It reads it all as one breathless line, rather than breaking them up into segments.

Consolidation
- [ ] The table headers read their labels twice (e.g. “Debtor, Debtor”). This is redundant and not needed. 
- [ ] Approval modal should be titled “Approve Case Consolidation.” Put a period at the end of the title. That should encourage a screen reader to read it correctly. We also may want to change the lead in sentence to “This will confirm the Joint Administration of the following cases:”. 

Transfers
- [x] Update helper text for case transfers. It should say “and enter the new court division and case number.” That reduces redundancy.

**Comment by fmaddenflx** [comment]
I previously thought the following was resolved, but have now discovered that it was not:

Region 02 H2 is read as “heading level 2, 2 items, Region, level 1, 02, level 1.” That should probably be “Heading level 2, Region 02”.
